test/vcr_python_wrapper.py

```diff
@@ -35,6 +35,12 @@ def body_json_l2_matcher(r1, r2):
         return r1.body == r2.body
 
 
+def domain_query_matcher(r1, r2):
+    if r1.path == '/api/smart_proxies' and r2.path == '/api/smart_proxies':
+        return [q for q in r1.query if q[0] != 'search'] == [q for q in r2.query if q[0] != 'search']
```

Comment:
Should we use sorted() here to ignore the param order?

Reply:
according to vcrpy docs it is ordered

> query: The parsed query string of the request. Sorted list of name, value pairs.

https://vcrpy.readthedocs.io/en/latest/advanced.html?highlight=query#advanced-features

Reply:
Cool. If vcr does that for us, we're good